shell/app/modules/msp/pages/micro-service/overview.tsx

```diff
@@ -120,54 +117,44 @@ const Overview = () => {
   }, [projectsList, projectStatistics]);
 
   const list = React.useMemo(() => {
-    return projectsListWithStatics.filter((item) => item.displayName.toLowerCase().includes(filterKey));
-  }, [projectsListWithStatics, filterKey]);
-
-  const columns: CardColumnsProps<MS_INDEX.IMspProject>[] = [
-    {
-      dataIndex: 'displayName',
-      colProps: {
-        className: 'flex items-center',
-      },
-      render: (displayName: string, { logo, desc, type }) => {
-        const { icon, color, tag } = iconMap[type];
-        return (
-          <>
-            <div className="w-14 h-14 mr-2">
-              {logo ? <img src={logo} width={56} height={56} /> : <ErdaIcon type={icon} size={56} />}
-            </div>
-            <div>
-              <p className="mb-0 font-medium text-xl leading-8">{displayName}</p>
-              <Tag className="mb-0.5 text-xs leading-5 border-0" color={color}>
-                {tag}
-              </Tag>
-              <div className="text-xs	leading-5 desc">{desc || i18n.t('no description yet')}</div>
-            </div>
-          </>
-        );
-      },
-    },
-    {
-      dataIndex: 'id',
-      colProps: {
-        className: 'flex items-center',
-      },
-      children: {
-        columns: metric.map((item) => ({
-          dataIndex: item.dataIndex,
-          colProps: {
-            span: 6,
+    return projectsListWithStatics
+      .filter((item) => item.displayName.toLowerCase().includes(filterKey))
+      .map((item) => {
+        const { logo, desc, displayName, type, relationship, id } = item;
+        return {
+          ...item,
+          logoURL: logo,
+          icon: logo ? undefined : iconMap[type].icon,
+          title: displayName,
+          description: desc || i18n.t('no description yet'),
+          tags: [
+            {
+              label: iconMap[type].tag,
+              color: iconMap[type].color,
+            },
+          ],
+          kvInfos: metric.map((t) => {
+            return {
+              key: t.name,
+              value: t.renderData(item),
+              compWapper: (comp: JSX.Element) => <>{comp}</>,
+            };
+          }),
```

Comment:
Why not let backend update the protocol data?

Reply:
this page is not  componentization protocol